FccCondor.md

```diff
@@ -0,0 +1,187 @@
+[]() Getting Started with HTCondor CLI
+======================================
+
+Contents:
+
+-   [HTCondor CLI](#htcondor-cli)
+    -   [Overview](#overview)
+    -   [Submission](#submission)
+        -   [1- The submit file](#1-submit-file)
+        -   [2- Changing the job's requirements](#2-job-req)
+        -   [3- Changing the job's software](#3-job-soft)
+   -   [Testing](#testing)
+   -   [Perspective](#perspective)
+
+
+[]() Overview
+-------------
+
+HTCondor is a management system for job submission. It has been chosen as the successor of LSF at CERN.
+
+As for LSF, you have to log in to an lxplus machine with your cern account if you want to access to the batch.
+
+
+[]() Submission
+---------------
+
+Instead of passing the job and its requirements as arguments to the command as you
+usually do with bsub, you just have to pass a submit file :
+
+	condor_q job.sub
+
+If you are bored to understand how it works you can directly play with HTcondor [here !](#testing) otherwise, please continue.
+
+### []() 1- The submit file
+
+First of all you have to write a submit file i.e **job.sub** as an example with the following content :
+
+	executable            = tmpjob.sh
+
+	arguments             = $(ClusterID) $(ProcId)
+
+	output                = output/job.$(ClusterId).$(ProcId).out
+
+	log                   = log/job.$(ClusterId).log
+
+	error                 = error/job.$(ClusterId).$(ProcId).err
+
+	send_credential        = True
+
+	queue
+
+
+where we have the following attributes :
+
+- executable is the path of your job
+- arguments are the arguments of your job
+- output contains the path of the standard output of your job
+- error contains the path of the standard error of your job
+
+
+### []() 2- Changing the job's requirements
+
+We provide you a simple submit file but generally, we have more complex stuff to do.
```

Comment:
"we have more complex stuff to do" -> you will want to add more options to your file.